pandas/core/arrays/datetimes.py

```diff
@@ -1011,6 +1011,30 @@ def date(self):
         'is_month_start',
         'is_month_start',
         "Logical indicating if first day of month (defined by frequency)")
+    """
+        Indicates whether the date is the first day of the month.
+        Returns
+        -------
+        Series of bool.
```

Comment:
Can you remove the period at the end, and add a description indented in the next line.